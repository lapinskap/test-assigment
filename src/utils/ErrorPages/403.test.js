import React from 'react';
import renderer from 'react-test-renderer';
import ForbiddenErrorPage from './403';
import '../../tests/setupTests';

describe('<ForbiddenErrorPage />', () => {
  afterEach(() => {
    jest.clearAllMocks();
  });

  it('renders correctly', () => {
    const tree = renderer.create(<ForbiddenErrorPage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
