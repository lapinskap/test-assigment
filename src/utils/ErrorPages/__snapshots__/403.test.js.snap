// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ForbiddenErrorPage /> renders correctly 1`] = `
<div
  className="vh-100 row"
>
  <div
    className="my-auto col"
  >
    <div
      className="text-center "
    >
      <i
        className="pe-7s-attention icon-gradient bg-plum-plate error-page-icon"
      />
      <div>
        Nie masz dostÄ™pu do tego zasobu
      </div>
    </div>
  </div>
</div>
`;
